platformio.ini

```diff
@@ -1435,6 +1435,7 @@ lib_ignore    = Adafruit NeoPixel
 build_type    = debug
 build_flags   = ${env:simulator_unix_release.build_flags}
   -ggdb -Og -D_THREAD_SAFE
+  -D"'section(name)='"
```

Comment:
What does this little trick do?

Reply:
Solves that u8glib font `.section` issue that gives error when compiling u8glib for Mac.

Reply:
I see. Does it work by replacing all instances of `'section(name)='` in the code with a blank string?

Reply:
Yes, this is exactly the same as:
```cpp
#define section(name)
```
To define complex macros using command line, we need put it in quotes: `-D'macro(foo)=foo+1'`. 
But PIO will remove the first level of quotes (I don't know why!!), so we need put two quotes... `-D"'macro(foo)=foo+1'"`...

Yes, a hack inside another hack. But it's fun! 😁

Reply:
Any reason someone might want to use a `section` directive in the future? This could drive them nuts.

Reply:
I think no one will never use `.section` in a native build...